python/README.rst

```diff
@@ -99,13 +95,13 @@ For Ubuntu/Debian/Mint
 
 .. code-block:: bash
 
-    $ apt-get install libboost-program-options-dev zlib1g-dev libboost-python-dev
+    $ apt-get install libboost-dev libboost-program-options-dev libboost-system-dev libboost-thread-dev libboost-math-dev libboost-test-dev libboost-python-dev zlib1g-dev cmake 
 
 For Mac OSX
 
 .. code-block:: bash
 
-    $ brew install libtool autoconf automake
+    $ brew install cmake
     $ brew install boost
     $ brew install boost-python
```

Comment:
From the wiki (that I added) 😈 
```
#If you want to build with python 3 support
brew install boost-python3
```

Reply:
Was already there, but I made the wording consistent with the wiki